### ISSUE TITLE
React native: unable to resolve module 'util' 

### ISSUE THREAD

**Issue by mfazekas** [Bootstragram/zealit#3]
```
Error: undefined Unable to resolve module `util` from `node_modules/zealit/zealit.js`: util could not be found within the project.
```

**Comment by mfazekas** [comment]
`npm install util --save` solves the issue in RN. Not sure if this package can anything to be more RN friendly

**Comment by mfazekas** [comment]
Also in RN we'd want to add 
```js
        '$$typeof',
        'prototype',
```

the the ignored properties, console.log and HMR seems to be accessing those.

**Comment by zertyuio** [comment]
Hi, thanks for this feedback.

About the `util` dependency, could you try this:
https://stackoverflow.com/questions/43055748/loading-modules-conditionally-in-react-native

In other words, update `zealit/package.json` in order to append:
```
"react-native": {
    "util": "~0.12.2"
}
```

Then `npm i`?
I don't have ReactNative on my computer, I can't do a quick test.

**Comment by mfazekas** [comment]
I've added a PR #4 that implements and alternative solution it will not require('util') on react native. 

I guess adding `util` as 'react-native' specific dependency works as well, but it's a bit pointless. As RN will not use `util.inspect.custom` for any inspection anyway.

I'll look into it if you prefer that to the conditional require in the `.js`

**Comment by zertyuio** [comment]
In fact, the PR will not work with nodejs, "navigator" is not defined in this context (at least v6.7.0). That's why I would like to explore the conditional depency.

If adding `util` is not good, could you try this ?
```
"react-native": {
    "util": false
}
```
If I understand stackoverflow response, `util` will look like an empty object.

**Comment by zertyuio** [comment]
If:
```
"react-native": {
    "util": false
}
```

doesn't work, I will try something else.

**Comment by mfazekas** [comment]
> In fact, the PR will not work with nodejs, "navigator" is not defined in this context (at least v6.7.0). That's why I would like to explore the conditional depency.

Yes but my pr, has other not only conditional include of util it also adds rn specific inspect properties to ignore. We can definetedly do ```(typeof(navigator) === 'object' && navigator !== null) && navigator.product === 'ReactNative'```, that should work everywhere.

**Comment by zertyuio** [comment]
Yes, I saw your PR, thanks for the proposal, I understand the issue. Before merging, i would like to explore the possibilities.

**Comment by mfazekas** [comment]
@zertyuio sure i'll test the util: false later today or tomorrow

**Comment by mfazekas** [comment]
I've tested:

```json
"react-native": {
    "util": "~0.12.2"
}
```

no util package was installed and same error on require


I've tested:

```json
"react-native": {
    "util": false
}
```

no util package was installed and `require('util')` returned `{}`

So the second one seems to be an option too.
